pkg/scheduler/factory.go

```diff
@@ -192,7 +192,6 @@ func (c *Configurator) create() (*Scheduler, error) {
 		c.nodeInfoSnapshot,
 		extenders,
 		c.informerFactory.Core().V1().PersistentVolumeClaims().Lister(),
-		GetPodDisruptionBudgetLister(c.informerFactory),
```

Comment:
we need to remove the function itself and check the feature flag and skip evaluating pdb if not set, we can do that as a followup.

Reply:
it needs to be done in this PR. As is, this PR fails the GA-only conformance e2e test